PreLoad.cmake

```diff
@@ -51,18 +51,32 @@ endif()
 execute_process(COMMAND uname -s OUTPUT_VARIABLE OS)
 execute_process(COMMAND uname -m OUTPUT_VARIABLE ARCH)
 
+# By default, prefer clang on Linux
```

Comment:
(not related but perhaps you want to add in l. 37 the remaining environment variables and flags checked in l. 23-27)

Reply:
Yeah, I already thought about this, thanks for the note, will do.